primer3/thermoanalysis.pyx

```diff
@@ -542,30 +676,47 @@ cdef class ThermoAnalysis:
         return ThermoAnalysis.calcEndStability_c(<ThermoAnalysis> self, s1, s2)
 
     cdef inline double calcTm_c(ThermoAnalysis self, char *s1):
+        '''
+        C only Tm computation
+
+        Args:
+            s1: sequence string 1
+
+        Returns:
+            floating point Tm result
+        '''
         cdef thal_args *ta = &self.thalargs
-        return seqtm(<const char*> s1,
-                     ta.dna_conc,
-                     ta.mv,
-                     ta.dv,
-                     ta.dntp,
-                     self.max_nn_length,
-                     <tm_method_type>
-                     self.tm_method,
-                     <salt_correction_type> self.salt_correction_method)
-
-    def calcTm(ThermoAnalysis self, seq1):
+        return seqtm(
+            <const char*> s1,
+            ta.dna_conc,
+            ta.mv,
+            ta.dv,
+            ta.dntp,
+            self.max_nn_length,
+            <tm_method_type>
```

Comment:
Did the formatter / linter place this on a separate line from `self.tm_method`? Different style than line 698.

Reply:
good catch